packages/flex-plugin-e2e-tests/src/utils/pages/index.ts

```diff
@@ -0,0 +1,96 @@
+import { existsSync, mkdirSync } from 'fs';
```

Comment:
`index.ts` files should usually just be the import/export - move this `App` class to  `pages/app.ts` and then just import export the app/base and all other classes from `pages/` in this file.